index.js

```diff
@@ -0,0 +1,233 @@
+  },
+  {
+    id: 675465,
+    title: "Fracture",
+  },
+  {
+    id: 70111470,
+    title: "Die Hard",
+  },
+  {
+    id: 654356453,
+    title: "Bad Boys",
+  },
+];
+
+const turnArrayIntoObj = (arr) => {
+  return arr.reduce((result, obj) => {
+    const id = obj.id;
+    const title = obj.title;
+    result[id] = title;
+    return result;
+  }, {});
+};
+
+console.log(turnArrayIntoObj(videos));
+
+// 3 Выведите массив ids для релизов у которых рейтинг 5.0
+const newReleases = [
+  {
+    id: 70111470,
+    title: "Die Hard",
+    boxart: "http://cdn-0.nflximg.com/images/2891/DieHard.jpg",
+    uri: "http://api.netflix.com/catalog/titles/movies/70111470",
+    rating: [4.0],
+    bookmark: [],
+  },
+  {
+    id: 654356453,
+    title: "Bad Boys",
+    boxart: "http://cdn-0.nflximg.com/images/2891/BadBoys.jpg",
+    uri: "http://api.netflix.com/catalog/titles/movies/70111470",
+    rating: [5.0],
+    bookmark: [{ id: 432534, time: 65876586 }],
+  },
+  {
+    id: 65432445,
+    title: "The Chamber",
+    boxart: "http://cdn-0.nflximg.com/images/2891/TheChamber.jpg",
+    uri: "http://api.netflix.com/catalog/titles/movies/70111470",
+    rating: [4.0],
+    bookmark: [],
+  },
+  {
+    id: 675465,
+    title: "Fracture",
+    boxart: "http://cdn-0.nflximg.com/images/2891/Fracture.jpg",
+    uri: "http://api.netflix.com/catalog/titles/movies/70111470",
+    rating: [5.0],
+    bookmark: [{ id: 432534, time: 65876586 }],
+  },
+];
+
+const getRelease = (arr) => {
+  return arr.map((release) => {
+    if (release.rating[0] === 5.0) {
+      return release.id;
+    }
+  });
+};
+// почему undefiend?
+console.log(getRelease(newReleases));
+
+// 4 С помощью функций map, reduce, вывести url у которого самая большая площадь
+
+const boxarts = [
+  {
+    width: 200,
+    height: 200,
+    url: "http://cdn-0.nflximg.com/images/2891/Fracture200.jpg",
+  },
+  {
+    width: 150,
+    height: 200,
+    url: "http://cdn-0.nflximg.com/images/2891/Fracture150.jpg",
+  },
+  {
+    width: 300,
+    height: 200,
+    url: "http://cdn-0.nflximg.com/images/2891/Fracture300.jpg",
+  },
+  {
+    width: 425,
+    height: 150,
+    url: "http://cdn-0.nflximg.com/images/2891/Fracture425.jpg",
+  },
+];
+
+const getBiggestArea = (arr) => {
+  const arrOfAreas = arr.map((item) => {
+    return item.width * item.height;
+  });
+  const maxArea = arrOfAreas.sort((a, b) => b - a)[0];
+
+  const objectUrlArea = arr.reduce((result, obj) => {
+    const url = obj.url;
+    const area = obj.width * obj.height;
+    result[url] = area;
+    return result;
+  }, {});
+
+  for (const key in objectUrlArea) {
+    if (objectUrlArea[key] === maxArea) {
+      return key;
+    }
+  }
+};
+
+console.log(getBiggestArea(boxarts));
+
+// 7
+
+const arr7 = [1, 2, 3, "1", "100", 7, "8", 9];
+
+const turnNumbersToString = (arr) => {
+  return arr.map((item) => {
+    if (typeof item === "number") {
+      return String(item);
+    } else if (typeof item === "string") {
```

Comment:
1. redundant `else` block.
when you have return at `if` block, you don't need the `else`, just because after return functin stops it's execution
2. you need `else if`  only in case when you have more than 2 options.